pkg/orderers/internal/parts/viewchange/pbftviewchange.go

```diff
@@ -627,6 +633,10 @@ func applyMsgNewView(
 		return
 	}
 
+	if len(newView.ViewChangeSenders) == 0 || len(newView.ViewChangeSenders) != len(newView.SignedViewChanges) {
+		return
```

Comment:
Why are we checking for 0 separately here? The second part of the check intuitively seems enough, assuming that some other part of the code checks that there are sufficiently many SignedViewChanges.

Also, let's log a warning here.

Reply:
You are right, the code will take a wild journey until it eventually stops [here ](https://github.com/filecoin-project/mir/blob/8f352300a01bb71d1bc05cb62a92e0e105ad0862/pkg/orderers/internal/parts/viewchange/pbftviewchange.go#L212), but that suffices.

Reply:
Done removed.